ecommerce/extensions/payment/processors/hyperpay.py

```diff
@@ -82,7 +86,8 @@ def get_extended_profile_field(account_details, field_name, default_value=None):
                 (
                     field.get('field_value', default_value) for field in account_details['extended_profile']
                     if field['field_name'] == field_name
-                )
+                ),
+                default_value
```

Comment:
Can you backport this to the other client branch as well?

Reply:
@giovannicimolin, yes, will do. But not right away though.

Reply:
@giovannicimolin, the other branch already [has](https://github.com/open-craft/ecommerce/blob/opencraft-release/juniper.3-paytabs/ecommerce/extensions/payment/processors/paytabs.py#L42-L48) this change. So I guess I just did a poor copy-paste job.